table_position:
  dual_table        : [2.5, 2.4, 0.0]
  movable_table_1200: [4.5, 1.9, 0.0]
  movable_table_1500: [5.5, 1.9, 0.0]
  movable_table_1800: [6.5, 1.9, 0.0]

shooter_valve_on_duration:
  dual_table_lower_shooter      : 0.5
  dual_table_upper_left_shooter : 0.5
  dual_table_upper_right_shooter: 0.5
  movable_table_1200_shooter    : 0.5
  movable_table_1500_shooter    : 0.5
  movable_table_1800_shooter    : 0.5